index.html

```diff
@@ -1,65 +1,207 @@
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+
+            <div class="card grid-item card-2">
+                <div class="card-image">
+                    <img src="./images/Snapshoot Portfolio 1.png">
+                </div>
+                <div class="card-info">
+                    <p class="project-name">Multi-Post Stories</p>
+                    <div class="project-info">
+                        <div class="company">CANOPY</div>
+                        <div class="dot"></div>
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
+                </div>
+            </div>
+
+            <div class="card grid-item">
+                <div class="card-image">
+                    <img src="./images/Snapshoot Portfolio 2.png">
+                </div>
+                <div class="card-info">
+                    <p class="project-name">Tonic</p>
+                    <div class="project-info">
+                        <div class="company">CANOPY</div>
+                        <div class="dot"></div>
+                        <div class="position">Back End Dev</div>
+                        <div class="dot"></div>
+                        <div class="year">2015</div>
+                    </div>
+                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="stack">
+                        <li class="lang">HTML</li>
+                        <li class="lang">CSS</li>
+                        <li class="lang">Javascript</li>
+                    </ul>
+                    <div class="see-project">
+                        See project
+                    </div>
```

Comment:
- [ ] Please use the `button` insteadd of `div` adopting the *button* tag over the *div* tag for interactive elements to ensure better accessibility, improved keyboard navigation, built-in button behaviours, and consistent styling across different browsers. Please incorporate this practice into your HTML coding for future projects and updates.